### ISSUE TITLE
Is constant observation necessary for 3D-DIC?

### ISSUE THREAD

**Issue by Wei-Meng515** [dicengine/dice#260]
Hi DICe,

I was wondering if a constant observation is required by stereo camera system for 3D-DIC application? 

I am working on a project needs long-term strain monitoring. If I set up the two cameras, calibrate them, take initial photos as my reference, then move the two cameras away. After a month or two when I come back, can I set up the two cameras at the same position (the camera orientations and positions might not be exactly same as the first time), do the calibration again and take new photos to compare them with the initial ones to obtain the accumulated strain map?

Thank you very much!

**Comment by Wei-Meng515** [comment]
> You will introduce some errors if you move the cameras after calibrating them, but I don't have a good way to say exactly how much. One thing you can do is as I think you are suggesting and have a constant object that you know the topography of which doesn't move during the test. That could serve as a baseline for the errors introduced by moving the camera system.

Thank you very much for the explanation! 

So the errors will be introduced as long as I use the initial images (before moving cameras) as reference even if I re-calibrate the cameras, am I correct?

**Comment by dicengine** [comment]
Yes, that is correct. It’s almost impossible to move cameras without impacting the calibration.
________________________________
From: Wei-Meng515 ***@***.***>
Sent: Friday, June 17, 2022 8:25:29 AM
To: dicengine/dice ***@***.***>
Cc: Turner, Daniel Zack ***@***.***>; Comment ***@***.***>
Subject: [EXTERNAL] Re: [dicengine/dice] Is constant observation necessary for 3D-DIC? (Issue #260)


You will introduce some errors if you move the cameras after calibrating them, but I don't have a good way to say exactly how much. One thing you can do is as I think you are suggesting and have a constant object that you know the topography of which doesn't move during the test. That could serve as a baseline for the errors introduced by moving the camera system.

Thank you very much for the explanation!

So the errors will be introduced as long as I use the initial images (before moving cameras) as reference even if I re-calibrate the cameras, am I correct?

**Comment by lixh1223** [comment]
DICE environment configuration tutorial

**Comment by lixh1223** [comment]
DICE environment configuration tutorial：https://youtu.be/AXPfkSY3NrY
